src/index.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { App } from './components/App'
```

Comment:
Configuration should be for path like `components/App `. We always start import path from the folders directly located in `src`.